bank.rb

```diff
@@ -0,0 +1,34 @@
+module Bank
+  def browser
+    @browser ||= Watir::Browser.new :chrome
+  end
+
+  def log_in
+    browser.goto('https://www.victoriabank.md/ru/')
+    browser.img(title: 'VB24_web.png').click
+    puts 'enter login'
+    login = gets.chomp.to_s
+    puts 'enter password'
+    password = gets.chomp.to_s
+    browser.text_field(class: 'username').set(login)
+    browser.text_field(name: 'password').set(password)
+    browser.button(text: 'Login').click
+    sleep 1
+  end
+
+  def acc_trans
+    ac_parsed = JSON.parse(File.read('account.json'))
```

Comment:
Why do you read any files here? There is no place in this task for such logic. Your code should generate only 1 file with output or print it on screen.